config.yaml

```diff
@@ -2,7 +2,11 @@
   product-listing-id: 63b85b573112fe5a95ee9a3a
   email-recipients:
     - openshift-ai-devtestops@redhat.com 
-- name: RHBAR
+- name: RHBOAR
   product-listing-id: 66697a7154e5fd53fd1ebdd2
   email-recipients:
-    - riprasad@redhat.com
\ No newline at end of file
+    - riprasad@redhat.com
+    - nipatil@redhat.com
+    - pantinor@redhat.com
+    - ewittman@redhat.com
+    - rkubis@redhat.com
```

Comment:
@NikishaPatil Can you removemy mail. It was just for the testing purpose.

You could either use individual mail-ids or use the team's mail like us. Just a suggestion

Reply:
Yes that is a better one to use the team's mail. I shall check for the mail id and update the details. 
Thank you.

Reply:
Team email isn't available yet , so I have updated all the email ids' for now .